### DISCUSSION TITLE
Will node red update in HA to 3.0?

### DISCUSSION THREAD

**Discussion by neil1111** [zachowj/hass-node-red#138] [Feature Requests]
Hi. I saw on YouTube that a major update to node red was just released. It includes many great new features. Here is the video link: [What’s New in Node-Red 3.0](https://youtu.be/R9bV2MytHr0)

is the version in HA scheduled to be updated, and if so, is there a timeline planned? 

Thanks, this is a fabulous and much appreciated integration!
Neil

**Comment by zachowj** [comment]
I'm guessing you're looking for the [hassio-addon](https://github.com/hassio-addons/addon-node-red) for NR.

**Reply by neil1111** [reply]
Ahh, thanks
